src/libraries/System.Collections/src/System/Collections/Generic/SortedDictionary.cs

```diff
@@ -220,7 +220,10 @@ public void Add(TKey key, TValue value)
 
         public void Clear()
         {
-            _set.Clear();
+            if (_set.Count != 0)
```

Comment:
Why add this check?

Reply:
If there are no elements then avoid the call to base Clear() method and just exit!

Reply:
Oh, if it's slow then I will remove this. My intention was that if the size is already 0, then calling clear won't make any sense, and just wanted to avoid that whole call to base Clear().